### PR TITLE
[MD][IP]Update button position & Add UT & Add skip option

### PR DESCRIPTION
**Pull request by kristenTian**
### Description
- Update button position per UX feedback for consistency
- Add UT
- Allow skip config datasource to use default os cluster
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/fixtures/stubbed_saved_object_index_pattern.ts [hunk 1]
```diff
@@ -44,5 +44,12 @@ export function stubbedSavedObjectIndexPattern(id: string | null = null) {
       title: 'title',
     },
     version: '2',
+    reference: [
+      {
+        id: 'id',
+        name: 'name',
+        type: 'data-source',
+      },
+    ],
```
Comment (zengyan-amazon) [inline src/fixtures/stubbed_saved_object_index_pattern.ts:53]: shall we add a parameter to this function to control if we want to add reference in the doc or not? we should test both multi-source enable and disable cases.

Comment (kristenTian) [reply src/fixtures/stubbed_saved_object_index_pattern.ts:53]: make sense

File: src/fixtures/stubbed_saved_object_index_pattern.ts [hunk 2]
```diff
@@ -33,8 +33,8 @@ import stubbedLogstashFields from './logstash_fields';
 
 const mockLogstashFields = stubbedLogstashFields();
 
-export function stubbedSavedObjectIndexPattern(id: string | null = null) {
-  return {
+export function stubbedSavedObjectIndexPattern(id: string | null = null, withDataSource?: boolean) {
```
Comment (zengyan-amazon) [inline src/fixtures/stubbed_saved_object_index_pattern.ts:36]: give `withDataSource` default value to false?

**Review by zengyan-amazon — APPROVED** [review]

File: src/fixtures/stubbed_saved_object_index_pattern.ts (hunk 2 above)
Comment (kristenTian) [reply src/fixtures/stubbed_saved_object_index_pattern.ts:36]: sure,to be more explicit

**Pull request merged**